File: imu_calib_python/imu_calib/calib/analyze_acc_allan.py
from __future__ import annotations

import numpy as np

from .analyze_allan_common import analyze_allan_common


def analyze_acc_allan(t: np.ndarray, acc: np.ndarray, **kwargs) -> dict:
    """Compute a basic Allan deviation estimate for accelerometer data."""
    return analyze_allan_common(t, acc, "acc", **kwargs)

File: imu_calib_matlab/src/analysis/analyze_acc_allan.m
function allan = analyze_acc_allan(t, acc, varargin)
%ANALYZE_ACC_ALLAN Compute a basic Allan deviation estimate for accelerometer data.
% Usage:
%   allan = analyze_acc_allan(t, acc)

allan = analyze_allan_common(t, acc, 'acc', varargin{:});
end
